lighthouse-treemap/app/src/main.js

```diff
@@ -0,0 +1,85 @@
+/**
+ * @license Copyright 2020 The Lighthouse Authors. All Rights Reserved.
+ * Licensed under the Apache License, Version 2.0 (the "License"); you may not use this file except in compliance with the License. You may obtain a copy of the License at http://www.apache.org/licenses/LICENSE-2.0
+ * Unless required by applicable law or agreed to in writing, software distributed under the License is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the License for the specific language governing permissions and limitations under the License.
+ */
+'use strict';
+
+/* eslint-env browser */
+
+/* globals webtreemap Util */
+
+/**
+ * Allows for saving the document and loading with data intact.
+ */
+function injectOptions() {
+  // @ts-expect-error
+  if (!window.__injected) return;
+
+  const scriptEl = document.createElement('script');
+  scriptEl.innerHTML = `
```

Comment:
any particular reason we needed to use `innerHTML`?

```suggestion
  scriptEl.textContent = `
```